site/src/lib/apiClient.ts

```diff
@@ -42,17 +42,32 @@ const axiosClient = axios.create({
   withCredentials: true,
 });
 
-const handleAxiosError = (
-  error: AxiosError,
-): { error: AxiosError<{ errors: Partial<ValidationError>[] }> } => {
-  /** @todo: stop special casing unauthorized errors */
-  if (error.response?.status === 401 || error.response?.data.errors) {
-    return { error };
-  }
-  throw error;
+export type ApiClientError = AxiosError<{
+  errors?: Partial<ValidationError>[];
+}> & { parsedErrorMessage: string };
+
+const parseErrorMessageFromAxiosError = (error: ApiClientError) => {
+  const firstValidationErrorMessage = error.response?.data.errors?.find(
+    ({ msg }) => !!msg,
+  )?.msg;
+
+  return (
+    firstValidationErrorMessage ??
+    error.response?.statusText ??
+    "An unknown error occurred"
```

Comment:
I think it's better to call this "An error occured".

Reply:
Sure, have addressed this as part of https://github.com/blockprotocol/blockprotocol/pull/214/commits/69ed2a49086c3b85210a4b726a76c8879823248d